.github/workflows/rust.yml

```diff
@@ -71,6 +65,36 @@ jobs:
           # we run tests using BitVec<u64,_> which doesn't.
           args: --all-features --target wasm32-unknown-unknown
 
+  no_std:
+    name: Check no_std build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+        # no_std version currently needs nightly due to error_in_core feature
+      - name: Install Rust nightly toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
As with `scale-bits`, I don't want this to be nightly only; we should keep everything working on stable :)

Reply:
My bad, I understood you wrong there. Fixed it by defining two `CustomError` types